wp-content/themes/access/controllers/programs.php

```diff
@@ -146,6 +151,22 @@ public function getIconClass() {
     return self::ICON_COLORS[$key];
   }
 
+  /**
+   * Get the itemtype for structure data
+   *
+   * @return  String  The itemtype
+   */
+
+  public function getItemScope() {
+    if ($this->status == 'covid-response') {
```

Comment:
@theasteve check to see if this works. `status` is either an array or false, never a string.

I think it would be `$this->status['type']`.